Add link to Slack's Advanced Formatting Documentation for messages (#593)
screwdriver-cd/screwdriver-cd.github.io: docs/user-guide/metadata.md

You can customize [notification](./configuration/settings.html#slack) messages with meta. Meta keys are different for each notification plugin.

> Slack requires special string formatting including a mention or channel link in the message. You can read more about the available options in the [Slack Documentation](https://api.slack.com/reference/surfaces/formatting#advanced).

Reviewer: Small suggestion
```suggestion
You will need special string formatting for mentions or channel links in Slack. You can read more about the available options in the [Slack Documentation](https://api.slack.com/reference/surfaces/formatting#advanced).
```

-- print the index
-- NOTE: index for meta.(get/set) purposes is 0-based, so subtract 1 from the array size
print(#array - 1)

> ```

Author: This file was edited directly in Github - I'm not sure what this diff is.
